geo/src/algorithm/triangulate_delaunay.rs

```diff
@@ -0,0 +1,642 @@
+use std::{fmt, fmt::Debug};
+
+use crate::coord;
+use crate::{BoundingRect, Coord, GeoFloat, Line, MultiPoint, Polygon, Triangle};
+
+pub const DEFAULT_SUPER_TRIANGLE_EXPANSION: f64 = 20.;
```

Comment:
needs docs, if pub

Reply:
I'm going to add the docs as part of the prep for upstream work.